pt-BR/mail/chrome/messenger/imapMsgs.properties

```diff
@@ -264,28 +264,28 @@
 
 ## @name IMAP_MOVE_FOLDER_TO_TRASH
 ## @loc None
-5055=Are you sure you want to move the selected folder into the Trash?
+5055=Tem certeza de que deseja mover a pasta selecionada para a Lixeira?
 
-# Status - no messages to download 
+# Status - no messages to download
 ## @name IMAP_NO_NEW_MESSAGES
 ## @loc None
 5056=Não há novas mensagens no servidor
 
 ## @name IMAP_DEFAULT_ACCOUNT_NAME
 ## @loc None
-5057=Mail para %S
+5057=E-mail para %S
 
 ## @name IMAP_DELETE_NO_TRASH
 ## @loc None
-5058=Deleting this folder is not undoable and will delete all of the messages it contains, and its sub-folders. Are you sure you still want to delete this folder?
+5058=Ao excluir esta pasta, não será possível desfazer a operação e todas as suas mensagens e subpastas excluídas. Tem certeza de que ainda deseja excluir esta pasta?
```

Comment:
CORRECTIONS NEEDED it should be 5058=Ao excluir esta pasta, não será possível desfazer a operação e todas as suas mensagens e subpastas serão também excluídas. Tem certeza de que ainda deseja excluir esta pasta?